projects/hostap/project.yaml

```diff
@@ -0,0 +1,8 @@
+homepage: "https://w1.fi/cvs.html"
+primary_contact: "<primary_contact_email>"
```

Comment:
Let me know if you have any suggestions for primary_contact. Shall I reach out to hostap developer?

Reply:
Yes, please, we need a maintainer here. `auto_ccs` is more flexible and may include other interested parties.